web/src/widgetry.ts

```diff
@@ -0,0 +1,369 @@
+// and should be stable. `InitOutput` on the other hand will vary from app to app.
```

Comment:
First sentence cut off partly?

Reply:
yikes, this comment got really mangled, thanks for noticing. I've taken the opportunity to rewrite it.